docs/concepts/package-types.md

```diff
@@ -0,0 +1,225 @@
+file and an entrypoint located in `src/index.tsx`, that typically `render`s a
+React component to the DOM, to work.
+
+Apps support pretty much all the functionalities from
+[Create React App v5](https://create-react-app.dev/docs/custom-templates), with
+some important differences:
+
+- [ejecting](https://create-react-app.dev/docs/available-scripts/#npm-run-eject)
+  is not supported
+- [template support](../concepts/templates.md) is integrated in the
+  [`modular add`](../commands/add.md) command. Modular templates are not
+  guaranteed to be compatible with CRA templates.
+- Source files are loaded with the more performant
+  [`esbuild-loader`](https://github.com/privatenumber/esbuild-loader) in the
+  Webpack configuration, and Babel plugins are not supported.
+- [Local proxies](https://create-react-app.dev/docs/proxying-api-requests-in-development/)
+  are supported, but [esbuild mode](../configuration.md) doesn't support the
+  `package.json` `proxy` field. The more flexible
+  [manual proxy configration](https://create-react-app.dev/docs/proxying-api-requests-in-development/#configuring-the-proxy-manually)
+  is supported in both Webpack and esbuild mode.
+
+Apps can be built using [`modular build`](../commands/build.md): the resulting
+output is an optimized site that can be served statically. All code (files in
+`src` plus external dependencies required in the code) are bundled in a single
+blob of code that can be split in different files.
+
+Apps can be started using [`modular start`](../commands/start.md): a developer
+server runs on port 3000, serving the app with an additional runtime layer that
+provides developer experience functionalities like hot reloading and on-screen
+error overlay.
+
+## ESM View
+
+Modular `esm-view`s are similar to `app`s with some essential differences: they
+are compiled as [ES
+Modules(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules)]
+and all their external `import`s are rewritten to point to an
+[ESM CDN](../esm-views/esm-cdn.md). Like apps, they are be built with the same
+Webpack or esbuild configuration and support the same functionalities. Unlike
+apps, they don't allow the user to specify a custom `index.html`, they don't
+include a `public` folder in the build and they expect their entrypoint
+(`src/index.tsx`) to not `render` to the DOM, but to export something, typically
+a React Component.
+
+Esm Views are Modular's type of choice to implement the
+[micro-frontend](../concepts/microfrontends.md) pattern. They typically expect
+their entrypoint exports to be
+[dynamically `import`ed at runtime](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import)
+by some other micro-frontend, but can be also
+[served standalone](../esm-views/how-to-build.md) exactly as an `app`, provided
+that they default export a React component that can be rendered to the DOM.
+
+Since the micro-frontend pattern allows different teams to build and serve their
```

Comment:
This paragraph is very wordy and I fear it will get lost. Is there a better way to provide this information? E.g. in a "How to use a CDN for external dependencies" guide or something?

Reply:
I fear we don't have such a resource. Do you wanna try to simplify this? I would like to convey the message that we have esm views because they can be built in isolation and composed at run-time.